tests/chains/ethereum/Dockerfiles/ganache/mnemonic

```diff
@@ -0,0 +1 @@
+math razor capable expose worth grape metal sunset metal sudden usage scheme
```

Comment:
Are you using this mnemonic somewhere?

Reply:
Remove unused file mnemonic
https://github.com/datachainlab/yui-relayer-build/commit/9ad5aca503830fc1d084dc839397f39de10ad8bb